stroyprombeton/models.py

```diff
@@ -41,13 +41,105 @@ def get_absolute_url(self):
         return reverse('category', args=(self.id,))
 
 
-class Product(catalog_models.AbstractProduct, page_models.PageMixin):
+class Option(models.Model):
+    """This doc page describes what is option: https://goo.gl/S4U9PG"""
+    tags = models.ManyToManyField(
+        'Tag',
+        related_name='options',
+        blank=True,
+        null=True,
+        verbose_name=_('tags'),
+    )
+    date_price_updated = models.DateField(auto_now_add=True, verbose_name=_('date price updated'))
```

Comment:
Will we move these fields into tags, after we will finish with tags' units?

Reply:
@artemiy312 , `data_price_updated` - no, because it's specific to Option.
But all other fields in this code block - yes. I'll split such fields and provide comment